### PR TITLE
Use pkg-config to find rpgp library

### PR DESCRIPTION
**Pull request by Jikstra**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -111,8 +111,8 @@ endif
 if get_option('rpgp')
   # The rpgp pkg-config file is currently incorrect, so just try
   # linking it instead for now.
-  #rpgp = dependency('rpgp')
-  rpgp = cc.find_library('rpgp')
+  rpgp = dependency('rpgp')
+  #rpgp = cc.find_library('rpgp')
```
Comment (hpk42) [inline meson.build]: shouldn't this remove the comment above?

Comment (Jikstra) [reply meson.build]: @flub any ideas?

Comment (flub) [reply meson.build]: yes, also no need to keep the cc.find_library(...) around in commented form, just delete it

Comment (flub) [reply meson.build]: but whatever, let's merge this since ppl already fall over this on irc.  we can remove comments afterwards

**Pull request merged**